ESPixelStick/src/output/OutputUart.hpp

```diff
@@ -39,26 +45,35 @@ class c_OutputUart
         OUTPUT_UART_8N2,
     };
 
-// TX FIFO trigger level. 40 bytes gives 100us before the FIFO goes empty
+// TX FIFO trigger level. WS2811 40 bytes gives 100us before the FIFO goes empty
 // We need to fill the FIFO at a rate faster than 0.3us per byte (1.2us/pixel)
-#define DEFAULT_UART_FIFO_TRIGGER_LEVEL (16)
+#define DEFAULT_UART_FIFO_TRIGGER_LEVEL (17)
+
+    enum TranslateIntensityData_t
```

Comment:
This would overload the term frame which in my case is used for the pixel / data frame starting at channel zero and going through the last channel data. Very few people even understand that the UART  level also has the concept of a frame. I am going to keep the current names.

Reply:
OK.  Thanks for considering my feedback.  I respect your choice.